src/domains/integration/service/integration.service.impl.ts

```diff
@@ -66,17 +68,13 @@ export class IntegrationServiceImpl implements IntegrationService {
             authorizedMemberMails,
             organizationId,
             issueProviderName,
-            pendingProjectId: pendingProject.id,
           }),
         { delay: 1000 * 60 * 60 * 2, retries: 3 },
         [PrismaClientKnownRequestError, PrismaClientUnknownRequestError, ConflictException]
       );
     } catch (e: any) {
-      Logger.error(e.message);
-      Logger.error(e.cause);
-      Logger.error(e.stack);
       await this.emailService.sendIntegrationErrorEmail(integratorEmail);
-      await this.pendingAuthProjectRepository.update({ pendingProjectId: pendingProject.id, isIntegrating: false });
```

Comment:
As, delete action is only performed logically, update method to change isIntegrating property should be made as well. It is to have consistent information about pending project. This modification was already made in [TRI-240 pull request](https://github.com/sirius-valley/tricker-backend/pull/108)